DangoPlop/Assets/Scripts/Ball_Behavioiur.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Ball_Behavioiur : MonoBehaviour {
+    public GameObject player;
+    public GameObject ball;
+    public Vector2 ballPos;
+    public Rigidbody2D rb;
+    public int thrust;
+    public CircleCollider2D circle;
+    public int counter;
+
+    // Use this for initialization
+    void Start () {
+        ball = GameObject.FindGameObjectWithTag("Ball");
```

Comment:
yes, but each different ball has its own properties (i.e position) so if we want, in the future, to use it, then we can
i can remove it if you want tho